### PR TITLE
Feature - User signup

### PR DESCRIPTION
**Pull request by iammiracle**
In this PR, you will find endpoints for creating, updating, deleting and querying all users.

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -35,4 +36,17 @@ router.get('/reports/:id', reportController.getOne);
...
 // Delete a Report with id
 router.delete('/reports/:id', reportController.deleteReport);
+
+// Create new users
+router.post('/user', userController.create);
+
+// Update a user with id
+router.put('/user/:id', userController.update);
+
+// Retrieve all reports
```
Comment (codemutatio) [inline server/routes/index.js]: It should be retrieve all users

**Comment by codemutatio** [comment]
Looks good to me, but should not be able to get an individual user?

File: server/routes/index.js (hunk 1 above)
Comment (iammiracle) [reply server/routes/index.js]: Yo..Done. Thanks Chief

**Comment by iammiracle** [comment]
So the route to query single user has been added

**Review by enyeInc — APPROVED** [review]
LGTM

**Pull request merged**